src/workerd/api/node/tests/zlib-nodejs-test.js

```diff
@@ -0,0 +1,215 @@
+import {
+  strictEqual,
+  throws,
+} from 'node:assert';
+import { Buffer } from 'node:buffer';
+import { crc32 } from 'node:zlib';
+
+
+// The following test data comes from
+// https://github.com/zlib-ng/zlib-ng/blob/5401b24/test/test_crc32.cc
```

Comment:
love the discussion on testcase sourcing, hope to see more of this in the future